main.go

```diff
@@ -430,33 +433,45 @@ func main() {
 
 	// APK
 	if len(signedAPKPaths) > 0 {
-		if err := tools.ExportEnvironmentWithEnvman("BITRISE_SIGNED_APK_PATH", signedAPKPaths[len(signedAPKPaths)-1]); err != nil {
-			log.Warnf("Failed to export Build Artifact, error: %s", err)
-		}
-		log.Donef("The Signed APK path is now available in the Environment Variable: BITRISE_SIGNED_APK_PATH (value: %s)", signedAPKPaths[len(signedAPKPaths)-1])
-
-		if err := tools.ExportEnvironmentWithEnvman("BITRISE_SIGNED_APK_PATH_LIST", joinedAPKOutputPaths); err != nil {
-			log.Warnf("Failed to export APK, error: %s", err)
-		}
-		log.Donef("The Signed APK path list is now available in the Environment Variable: BITRISE_SIGNED_APK_PATH_LIST (value: %s)", joinedAPKOutputPaths)
+		exportAPK(signedAPKPaths, joinedAPKOutputPaths)
 	} else {
 		log.Debugf("No Signed APK was exported - skip BITRISE_SIGNED_APK_PATH Environment Variable export")
 		log.Debugf("No Signed APK was exported - skip BITRISE_SIGNED_APK_PATH_LIST Environment Variable export")
 	}
 
 	// AAB
 	if len(signedAABPaths) > 0 {
-		if err := tools.ExportEnvironmentWithEnvman("BITRISE_SIGNED_AAB_PATH", signedAABPaths[len(signedAABPaths)-1]); err != nil {
-			log.Warnf("Failed to export Build Artifact, error: %s", err)
-		}
-		log.Donef("The Signed AAB path is now available in the Environment Variable: BITRISE_SIGNED_AAB_PATH (value: %s)", signedAABPaths[len(signedAABPaths)-1])
-
-		if err := tools.ExportEnvironmentWithEnvman("BITRISE_SIGNED_AAB_PATH_LIST", joinedAABOutputPaths); err != nil {
-			log.Warnf("Failed to export AAB, error: %s", err)
-		}
-		log.Donef("The Signed AAB path list is now available in the Environment Variable: BITRISE_SIGNED_AAB_PATH_LIST (value: %s)", joinedAABOutputPaths)
+		exportAAB(signedAABPaths, joinedAABOutputPaths)
 	} else {
 		log.Debugf("No Signed AAB was exported - skip BITRISE_SIGNED_AAB_PATH Environment Variable export")
 		log.Debugf("No Signed AAB was exported - skip BITRISE_SIGNED_AAB_PATH_LIST Environment Variable export")
 	}
 }
+
+func exportAPK(signedAPKPaths []string, joinedAPKOutputPaths string) {
```

Comment:
NOTE: if the only difference in the two method is `APK` and `AAB` then you can have a common method to handle both but switching only the extension.

Reply:
Yeah, I just wanted to avoid using more nested if. 🙃 
Currently, it's code duplication, but with it, you can much faster understand what happens, and you don't need to figure out the multiple if-else statements.